jerry-core/ecma/builtin-objects/ecma-builtin-bigint-prototype.c

```diff
@@ -125,6 +137,50 @@ ecma_builtin_bigint_prototype_object_to_string (ecma_value_t this_arg, /**< this
   return ecma_make_string_value (string_p);
 } /* ecma_builtin_bigint_prototype_object_to_string */
 
+/**
+ * Dispatcher of the built-in's routines
+ *
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value.
+ */
+ecma_value_t
+ecma_builtin_bigint_prototype_dispatch_routine (uint8_t builtin_routine_id, /**< built-in wide routine
+                                                                              *   identifier */
+                                                ecma_value_t this_arg, /**< 'this' argument value */
+                                                const ecma_value_t arguments_list_p[], /**< list of arguments
+                                                                                      *   passed to routine */
+                                                uint32_t arguments_number) /**< length of arguments' list */
+{
+  ecma_value_t this_value = ecma_builtin_bigint_prototype_object_value_of (this_arg);
+
+  if (ECMA_IS_VALUE_ERROR (this_value))
+  {
+    return this_value;
+  }
+
+  if (builtin_routine_id == ECMA_BIGINT_PROTOTYPE_VALUE_OF)
+  {
+    return ecma_copy_value (this_value);
+  }
+
+  switch (builtin_routine_id)
+  {
+    case ECMA_BIGINT_PROTOTYPE_TO_STRING:
+    {
+      return ecma_builtin_bigint_prototype_object_to_string (this_arg, arguments_list_p, arguments_number);
+    }
+    case ECMA_BIGINT_PROTOTYPE_TO_LOCALE_STRING:
+    {
+      return ecma_builtin_bigint_prototype_object_to_string (this_arg, 0, 0);
+    }
+    default:
+    {
+      JERRY_UNREACHABLE ();
+    }
+  }
+  ecma_free_value (this_value);
```

Comment:
This code is never executed. You should save the results of each routine call, free this arg, and return with the saved value.